services/community-site/src/utils/validators.ts

```diff
@@ -0,0 +1,34 @@
+import { Validator, ValidatorWithStats, YieldedValidator } from '../interfaces/Validator';
```

Comment:
I see this as important business logic, not a "utility" function.

Reply:
I added it inside utils because we don't have any other dedicated place for such business logic files inside the current folder tree as in the services we have only hooks. Should I create a new one and reorganize these?